### ISSUE TITLE
Add ability to highlight spoken text

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#14200]
### Is your feature request related to a problem? Please describe.
Vision Highlight puts a border around the current object or element.  It would be helpful for some users if there was a way to highlight the text being read by NVDA.

### Describe the solution you'd like

A highlight option to outline the current word when moving by word, or line if moving by line.

### Describe alternatives you've considered

One option which would need to be considered is "Say all" but highlighting either the current word or line being read would work for that.

**Comment by Brian1Gaff** [comment]
I remember my old copy of Supernova used to be able to do this, but often 
its reading speed was a  little slower, but then this was in Windows XP days 
and the hardware was less capable.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "Quentin Christensen" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Friday, September 30, 2022 7:51 AM
Subject: [nvaccess/nvda] Add ability to highlight spoken text (Issue #14200)

**Comment by Adriani90** [comment]
@Qchristensen is this use case somehow already tackled by the new magnifier?

**Comment by Qchristensen** [comment]
No, the magnifier only enlarges the whole screen - while it is centred around the current focus, it doesn't highlight it any more than anything else.  If anything, the magnifier potentially makes this more likely to be requested (by magnification users.
